Readme.md

```diff
@@ -6,18 +6,19 @@
 <!-- default file list -->
 *Files to look at*:
 
-* [Form1.cs](./CS/CustomDraw/Form1.cs) (VB: [Form1.vb](./VB/CustomDraw/Form1.vb))
+* [Form1.cs](./CS/PDF%Viewer/Form1.cs) (VB: [Form1.vb](./VB/PDF%Viewer/Form1.vb))
 <!-- default file list end -->
-# How to custom draw in  PDF Viewer
+# How to: Draw Graphics by Coordinates Obtained from the PDF Viewer
 
+The [PDF Viewer](https://www.devexpress.com/products/net/controls/winforms/pdf-viewer/) allows you to draw graphics in a PDf file in the [Control.Paint](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.paint) event handler. Use the [PDF Graphics API](https://docs.devexpress.com/OfficeFileAPI/119009/pdf-document-api/pdf-graphics) to save graphics to a file.
 
-The [PDF Viewer control](https://www.devexpress.com/products/net/controls/winforms/pdf-viewer/) can draw graphics in the PDF document in the <strong>Control.Paint</strong> event handler. <br><br>In this example, the filled rectangle is drawn at any document space when you hold down the left mouse button and move it. 
+> **IMPORTANT**
+> You need a license for the DevExpress [Office File API Subscription](https://www.devexpress.com/products/net/office-file-api/) or [DevExpress Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml) to use PDF Graphics API in production code.
```

Comment:
Isn't a license and a subscription two different things? By that I mean I'm not sure you can have a license for a subscription - you can either have one or the other.

Reply:
We use this template in many places, I never thought about it, to be honest. In the documentation, in blogs... seems like this phrasing is ok. There is a Subscription License Agreement, so you can have a license for a subscription.

Reply:
Ok